repository-data/webfiles/src/main/resources/site/less/demo.less

```diff
@@ -1,303 +1,458 @@
-body {
-    color: #425563;
-    font: 16px Arial;
-    letter-spacing: .025em;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    padding: 0;
-    margin: 0;
-}
 
 
 
 
-.doc-title {
-    font-size: 2.2em;
-    font-weight: 400;
-    margin: 12px 0 12px 0;
-}
 
 
+@import "normalise";
+@import "fonts";
+
+
+
+
+/* VARS */
+/* ########################################################## */
+@max-width: 1170px;
+@color-primary: #005EB8;
+@color-secondary: #00376D;
+@color-grey-dark: #3E4F59;
+@color-grey-light: #DEE2E5;
+
+
+
+@base-spacing-unit: 20px;
+@half-spacing-unit: @base-spacing-unit/2;
+@quarter-spacing-unit: @base-spacing-unit/4;
+@double-spacing-unit: @base-spacing-unit*2;
+
+@base-radius: 5px;
 
 
-/*
-<div class="container">
-</div>
- */
 .container {
-    width: 1200px;
+    width: @max-width;
```

Comment:
As discussed, can we add some qualifier to this, renaming it to something like 'document-content'?

Reply:
`container` is pretty well defined in Frontend Land. I would be surprised not to find "container" class tbh.